hammett/core/__init__.py

```diff
@@ -48,6 +48,7 @@ def __init__(
         persistence: 'BasePersistence[UD, CD, BD] | None' = None,
         states: 'States | None' = None,
         job_queue_handlers: 'list[dict[str, Any]] | None' = None,
+        error_handler: 'Callable[[Update, CallbackContext[BT, UD, CD, BD]], Any] | None' = None,
```

Comment:
- Rename it to `error_handlers` (by analogy with `job_queue_handlers`) with type of `list[Handler]`.
- Create `_register_error_handlers` by analogy with `_register_job_queue_handler`.